apps/experiments/tests/test_views.py

```
@@ -396,3 +401,27 @@ def test_get_queryset(self, experiment):
         view = ExperimentTableView()
         view.request = request
         assert list(view.get_queryset().all()) == [experiment]
+
+
+@pytest.mark.django_db()
```

Ruff

Use `@pytest.mark.django_db` over `@pytest.mark.django_db()`

Remove parentheses

(PT023)